src/App.jsx

```diff
@@ -1,104 +1,19 @@
 import './App.scss';
```

Comment:
The task requires that styles from `App.scss` be moved to separate files for each component. Please ensure that the `App.scss` file is split accordingly and removed if no longer needed.